config/config.cpp

```diff
@@ -12,20 +12,19 @@
 using namespace std;
 
 namespace config {
-int use_qat_compress = 1;
-int use_qat_uncompress = 1;
-int use_iaa_compress = 0;
-int use_iaa_uncompress = 0;
-int use_zlib_compress = 1;
-int use_zlib_uncompress = 1;
-int iaa_compress_percentage = 50;
-int iaa_uncompress_percentage = 50;
-int iaa_prepend_empty_block = 0;
-int qat_periodical_polling = 0;
-int qat_compression_level = 1;
+
 std::string log_file = "";
-int log_level = 2;
-int log_stats_samples = 1000;
+
+const std::string config_names[CONFIG_MAX]{
+    "use_qat_compress",        "use_qat_uncompress",
+    "use_iaa_compress",        "use_iaa_uncompress",
+    "use_zlib_compress",       "USE_ZLIB_UNCOMPRESS",
+    "use_zlib_uncompress",     "iaa_compress_percentage",
+    "iaa_prepend_empty_block", "qat_periodical_polling",
+    "qat_compression_level",   "log_level",
+    "log_stats_samples"};
+
+uint16_t configs[CONFIG_MAX] = {1, 1, 0, 0, 1, 1, 50, 50, 0, 0, 1, 2, 1000};
```

Comment:
I was wondering how to make the initialization more readable. Maybe we can have the corresponding option description as comment next to each value? 

```
uint16_t configs[CONFIG_MAX] = {
   1, /* use_qat_compress */
   1, /* user_qat_uncompress */
```
....